### ISSUE TITLE
Build Failing 

### ISSUE THREAD

**Issue by sreerammc** [Sitecore/docker-images#458]
I tried to call 

**_.\Build.ps1 -OSVersion @("2004", "1909", "ltsc2019") -IncludeSpe -IncludeSxa -IncludeJss -SkipExistingImage_**

I get following error

**scr.sitecore.com/sxp/sitecore-xp1-solr-init:10.0.0-ltsc2019
11:29:35:844: External image 'scr.sitecore.com/sxp/sitecore-xp1-solr-init:10.0.0-ltsc2019' is latest.
1909: Pulling from windows/servercore
docker : no matching manifest for windows/amd64 10.0.17763 in the manifest list entries
At C:\docker-images-master\build\modules\SitecoreImageBuilder\1.0.0\Public\Invoke-Build.ps1:113 char:17
'+                 docker image pull $tag
'+                 ~~~~~~~~~~~~~~~~~~~~~~
    '+ CategoryInfo          : NotSpecified: (no matching man...st list entries:String) [], RemoteException
    '+ FullyQualifiedErrorId : NativeCommandError**

- [x]  I have tried with the latest version of Docker for Windows (Stable or Edge) or Docker Enterprise.
- [x]  Windows version is at least 1809 / ltsc2019.
- [x] Windows has the latest updates installed.

**Comment by sreerammc** [comment]
When I try with following:

**.\Build.ps1 -InstallSourcePath C:\docker-images-master\build\packages -OSVersion @("ltsc2019") -IncludeSpe -IncludeSxa -IncludeJss -SkipExistingImage

I encounter and error saying:

WARNING: 13:24:25:334: Optional source file 'C:\docker-images-master\build\packages\Sitecore Connect for Salesforce CRM 5.0.0 rev. 01431.scwdp.zip' is missing...
Sending build context to Docker daemon  7.215kB

Step 1/17 : ARG BASE_BUILD_IMAGE
Step 2/17 : ARG TOOL_IMAGE
Step 3/17 : ARG BASE_IMAGE
Step 4/17 : ARG ASSETS_IMAGE
Step 5/17 : FROM ${TOOL_IMAGE} as tools
 ---> 089849b3d870
Step 6/17 : FROM ${BASE_BUILD_IMAGE} as build
 ---> 32fecabef723
Step 7/17 : SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 ---> Using cache
 ---> de84f43596bf
Step 8/17 : ARG MODULE_NAME
 ---> Using cache
 ---> c9a1826e3609
Step 9/17 : ARG ROLES
 ---> Using cache
 ---> 269bc2d36c00
Step 10/17 : COPY tools/ tools/
 ---> Using cache
 ---> d45201076282
Step 11/17 : COPY *.zip /wdp/
docker : COPY failed: no source files were specified
At line:1 char:1
'+ docker image build --isolation 'hyperv' --build-arg BASE_IMAGE=mcr.mi ...
'+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    '+ CategoryInfo          : NotSpecified: (COPY failed: no... were specified:String) [], RemoteException
    '+ FullyQualifiedErrorId : NativeCommandError
------------------------------------------------------------------------------------------------------------------------------

**I saw few warning saying 

WARNING: 13:23:49:539: Required package 'Sitecore Connect for Salesforce CRM 5.0.0 rev. 01431.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for Salesforce CRM 5.0.0 rev. 01431.scwdp.zip' into 'C:\docke
r-images-master\build\packages' manually.
WARNING: 13:23:49:541: Required package 'Sitecore Connect for Microsoft Dynamics 365 for Sales 5.0.0 rev. 01439.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for Microsoft Dynamics 365 for Sales 5.0.0 
rev. 01439.scwdp.zip' into 'C:\docker-images-master\build\packages' manually.
WARNING: 13:23:49:542: Required package 'Sitecore Connect for Salesforce Marketing Cloud - Content Exchange 5.0.0 rev. 00162.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for Salesforce Marketing Cloud
 - Content Exchange 5.0.0 rev. 00162.scwdp.zip' into 'C:\docker-images-master\build\packages' manually.
WARNING: 13:23:49:544: Required package 'Sitecore Connect for Salesforce Marketing Cloud - Behavioral Data Exchange 5.0.0 rev. 00294.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for Salesforce Marketi
ng Cloud - Behavioral Data Exchange 5.0.0 rev. 00294.scwdp.zip' into 'C:\docker-images-master\build\packages' manually.
13:23:49:545: Required package found: 'C:\docker-images-master\build\packages\Sitecore Connect for Salesforce Marketing Cloud - BDE Plugin for Tenant Service 5.0.0 rev. 00294.scwdp.zip'
WARNING: 13:23:49:545: Required package 'Sitecore Connect for CMP XP for 10.0 v. 3.0.0 rev. 00134.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for CMP XP for 10.0 v. 3.0.0 rev. 00134.scwdp.zip' into '
C:\docker-images-master\build\packages' manually.
WARNING: 13:23:49:545: Required package 'Sitecore Publishing Module 10.0.0.0 rev. r00568.2697.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Publishing Module 10.0.0.0 rev. r00568.2697.scwdp.zip' into 'C:\docke
r-images-master\build\packages' manually.
WARNING: 13:23:49:550: Required package 'Sitecore Connect for CMP XM for 10.0 v. 3.0.0 rev. 00134.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for CMP XM for 10.0 v. 3.0.0 rev. 00134.scwdp.zip' into '
C:\docker-images-master\build\packages' manually.
WARNING: 13:23:49:551: Required package 'Data Exchange Framework 5.0.0 rev. 01466.scwdp.zip' not available for download because the url property is empty, please copy 'Data Exchange Framework 5.0.0 rev. 01466.scwdp.zip' into 'C:\docker-images-master\build\pa
ckages' manually.
13:23:49:553: Required package found: 'C:\docker-images-master\build\packages\Sitecore Data Exchange Framework Action Plugin for Marketing Automation 5.0.0 rev. 01466.scwdp.zip'
WARNING: 13:23:49:555: Required package 'Sitecore Connect for Sitecore DAM-2.0.0.scwdp.zip' not available for download because the url property is empty, please copy 'Sitecore Connect for Sitecore DAM-2.0.0.scwdp.zip' into 'C:\docker-images-master\build\pack
ages' manually.

** May I know, where I can get these zip files?

**Comment by vtml** [comment]
Some of the scwdp.zip files are not available. I personally convert those myself with the Sitecore Azure Toolkit just to make things easier.

**Comment by sreerammc** [comment]
Whom should I bribe to get my hands on them? :) 

Any "how to guide" to create these?

**Comment by jeanfrancoislarente** [comment]
The `Download-Module-Prerequisites.ps1` should be able to handle the download and convert. It may download a few things you don't need but at least it'll get you moving forward.

In the very near future, we won't need to build some of these asset images as they'll come directly from Sitecore. We also need to potentially log a separate issue for improving the asset image build... `-IncludeModuleAssets` perhaps in build.ps1 which will skip the asset images if not specified.

**Comment by jeanfrancoislarente** [comment]
You can try with this branch @sreerammc .

Also, are you aware that the v10 images from this repo are **already pre-built and in the scr.sitecore.com registry**?

You can see a full list of tags in [sitecore-tags.md](https://github.com/Sitecore/docker-images/blob/master/tags/sitecore-tags.md). All module images are in `community/modules` and all pre-built base images are in `community`

ie: `docker pull scr.sitecore.com/community/sitecore-xm1-custom-sxa-cm:10.0.0-ltsc2019`

**Comment by AmitKumar-AK** [comment]
@vtml I tried downloading Sitecore Azure Tool kit but not succeed.

![image](https://user-images.githubusercontent.com/28383403/100451421-66785580-30dd-11eb-83a0-a72091a87334.png)


ConvertTo-SCModuleWebDeployPackage : The term 'ConvertTo-SCModuleWebDeployPackage' is not recognized as the name of a cmdlet, function, script file, or operable 
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ ConvertTo-SCModuleWebDeployPackage -Path "C:/Users/sitecore_admin/Dow ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ConvertTo-SCModuleWebDeployPackage:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by vtml** [comment]
@AmitKumar-AK you will need to follow the answer from this StackExchange https://sitecore.stackexchange.com/questions/10417/how-to-create-web-deploy-package-for-sitecore-modules

**Comment by AmitKumar-AK** [comment]
@vtml I am only getting following commands, not all:

![image](https://user-images.githubusercontent.com/28383403/100452579-6a0cdc00-30df-11eb-931e-33a1c3254907.png)

**Comment by jeanfrancoislarente** [comment]
`Import-Module .\tools\Sitecore.Cloud.Cmdlets.dll -Force`

Did you see [my comment](https://github.com/Sitecore/docker-images/issues/458#issuecomment-734814797) @AmitKumar-AK  about the images already available in the registry?

**Comment by AmitKumar-AK** [comment]
I am getting error as below:

Import-Module : Could not load file or assembly 'file:///C:\Users\sitecore_admin\Downloads\Sitecore Azure Toolkit 2.5.0-r02519.1061\tools\Sitecore.Cloud.Cmdlets.dll' or 
one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
At line:1 char:1
+ Import-Module .\tools\Sitecore.Cloud.Cmdlets.dll -Force
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Import-Module], FileLoadException
    + FullyQualifiedErrorId : System.IO.FileLoadException,Microsoft.PowerShell.Commands.ImportModuleCommand

**Comment by jeanfrancoislarente** [comment]
Did you pull the branch for the PR @AmitKumar-AK, it's in branch `feature/exclude-module-assets` #459

**Comment by AmitKumar-AK** [comment]
After some inputs from @Mohammad Abujaffal, if we unblock the DLL then only you can import.

**Comment by sanjeev-epam** [comment]
Same issue repeating, can you confirm this works in all the latest versions too?

**Comment by AmitKumar-AK** [comment]
@sanjeev-epam  if we unblock the DLL then only you can import.
